kratos/tests/test_gid_io.py

```diff
@@ -58,8 +58,8 @@ def __InitialRead(self):
 
     def __Check(self,output_file,reference_file):
         import filecmp
-        self.assertTrue( filecmp.cmp(os.path.dirname(os.path.realpath(__file__)) + "\\" 
-            +output_file, os.path.dirname(os.path.realpath(__file__)) + "\\" +reference_file))
+        self.assertTrue( filecmp.cmp(os.path.dirname(os.path.realpath(__file__)) + "/" 
+            +output_file, os.path.dirname(os.path.realpath(__file__)) + "/" +reference_file))

```

Comment:
I am afraid this won't work on win, but we are not testing it there yet so no problem, we have to implement a os-agnostic mechanism to parse paths some day.

Reply:
Do you have a suggestion how to do it?
Then I can implement it in the file-comparison process

Reply:
portable way to do this would be [os.path.join()](https://docs.python.org/3/library/os.path.html#os.path.join)